src/config.rs

```diff
@@ -3,6 +3,8 @@ use {
   clap::{App, AppSettings, Arg, ArgGroup, ArgMatches, ArgSettings},
 };
 
+pub(crate) const UNSTABLE_KEY: &str = "JUST_ALLOW_UNSTABLE";
```

Comment:
What about `JUST_UNSTABLE` instead? `JUST_UNSTABLE=true` makes sense, and is shorter.